### DISCUSSION TITLE
ssr config set to false

### DISCUSSION THREAD

**Discussion by gran0123** [ptkdev-boilerplate/svelte-kit-ssr-boilerplate#4] [General]
From what I've seen in your boilerplate, ssr is set to false in svelte.config.js where they state in their documentation that ["Enables or disables server-side rendering app-wide."](https://kit.svelte.dev/docs#configuration-ssr). So from that, this boilerplate is not ssr. If your are not hiding it anywhere else.

Another improvement that came to my mind is that this boilerplate doesn't have https support out-of-the-box since when we add `'-H'` flag to command, Vite can't connect as well when I checked in the web browser console. Continuous refreshing.

**Comment by ptkdev** [comment]
Hi @tobiasgranlof !
Thank's for your comments. SSR to false is a missprint, svelte-kit is an beta status and this boilerplate is a WIP and unstable. I use this for tests and try update it montly with latest svelte-kit@next but  is hard (svelte kit rewrite configs and functions continuously in beta version). Better wait for v1.0 stable of svelte-kit. 

All pull request with improvement are welcome.

**Reply by gran0123** [reply]
Yea no problem. I'm currently configuring this boilerplate for running https locally. Got it working and will make a pull request 👍

**Reply by ptkdev** [reply]
Thank's bro!
